auth-web/src/components/auth/MaintainBusinessInfo.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+  <v-container id="maintain-info-container">
+    <v-row>
+      <!-- Info Column -->
+      <v-col cols="12" md="6">
+        <h2>Manage and Maintain Your Business</h2>
+        <v-list-item class="list-item" v-for="(item, index) in bulletPoints" :key="index">
+          <v-icon size="6" class="list-item-bullet">mdi-square</v-icon>
+          <v-list-item-content>
+            <v-list-item-subtitle class="list-item-text">
+              {{item.text}}
+            </v-list-item-subtitle>
+          </v-list-item-content>
+        </v-list-item>
+        <!-- Sub Bullet Points -->
+        <v-list-item class="list-item list-item-sub" v-for="(item, index) in subBulletPoints" :key="`Sub-index: ${index}`">
+          <v-icon size="6" class="list-item-bullet">mdi-square</v-icon>
+          <v-list-item-content>
+            <v-list-item-subtitle class="list-item-text">
+              {{item.text}}
+            </v-list-item-subtitle>
+          </v-list-item-content>
+        </v-list-item>
+        <!-- Panel Btns -->
+        <div class="maintain-info-btns">
+          <v-btn v-if="userProfile" large color="#fcba19" class="my-5"
+            @click="emitManageBusinesses()">
+            Manage an Existing Business
+          </v-btn>
+          <v-btn v-else large color="#fcba19" @click="login()" class="my-5">
+            Log in with BC Services Card
+          </v-btn>
+          <v-btn large outlined color="#003366" class="btn-learn-more"
+            href="https://smallbusinessbc.ca/article/how-to-choose-the-right-business-structure-for-your-small-business/%7D"
+            target="_blank" rel="noopener noreferrer">
+            Learn More
+          </v-btn>
+        </div>
+      </v-col>
+      <!-- Image Column -->
+      <v-col cols="12" md="6">
+        <v-img src="../../assets/img/Step4-Maintain-x1.png" aspect-ratio="1.2" contain></v-img>
+      </v-col>
+    </v-row>
+  </v-container>
+</template>
+
+<script lang="ts">
+import { Component, Emit, Prop, Vue } from 'vue-property-decorator'
+import { Pages } from '@/util/constants'
+
+@Component({})
+export default class MaintainBusinessInfo extends Vue {
+  private bulletPoints: Array<any> = [
+    { text: 'Once your business is incorporated or registered you are required to keep information about your business up to date with the Registry.' },
+    { text: 'By managing your business throught your BC registry account you can:' }
```

Comment:
Typo: throught

Also should be "BC Registry" ?

Reply:
In next component below there's a reference to Business Registry. I'm confused, where are we referring to which? And is it singular or plural (registries)?

Reply:
Text has been fixed!
I don't understand the next comment however, which line specifically is that referring too?

Reply:
This line says "BC Registry". In another component (in this PR) it says "Business Registry". Are we using both? And is it Registry or Registries?

Reply:
kk, following up with Tracey 👍